common/src/main/resources/assets/wynntils/lang/en_us.json

```
@@ -3141,7 +3159,10 @@
   "screens.wynntils.settingsScreen.name": "Wynntils Settings",
   "screens.wynntils.settingsScreen.needToSelect": "You need to select a setting to perform this action!",
   "screens.wynntils.settingsScreen.next": "Next",
+  "screens.wynntils.settingsScreen.noFeaturesFound": "No features found matching your current filters, try changing the enable state filter or refining your search query.",
```

```suggestion
  "screens.wynntils.settingsScreen.noFeaturesFound": "No features found matching your current filters, try changing the enabled state filter or refining your search query.",
```

is it not "enabled" state filter?